### DISCUSSION TITLE
Should show path import error in Vs code editor Window

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [vercel/next.js#46929] [Help] [answered]
###  I can see error in Terminal but not in Editor Window . This is Strange Right ? 
![WrongPathExample](https://user-images.githubusercontent.com/71011043/223756832-2d44a130-47bc-47d4-ab9d-c387846c5579.png)

https://github.com/vitejs/vite/discussions/12338#discussion-4941825

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This is not a Vite, nor a VSCode forum, though I am more than glad to help out!

Create a `jsconfig.json` file at the root of your project, and place this in:

```json
{
    "compilerOptions": {
        "checkJs": true,
        "jsx": "react-jsx"
    }
}
```

**Reply by thanhtutzaw** [reply]
Thanks you It works . But Some js file didn't work . May be I need to restart vs code .
